src/tools/cephfs/top/cephfs-top

```diff
@@ -271,7 +269,7 @@ class FSTop(FSTopBase):
         self.client_name = args.id
         self.cluster_name = args.cluster
         self.conffile = args.conffile
-        self.refresh_interval_secs = args.delay
+        self.refresh_interval_secs = args.delay * 10
```

Comment:
There's no need to increase delay in functions like `set_key`, `choose_field` etc. A minimal delay is set there just to make the input mechanism to work correctly. But you can multiply in `run_display` and `run_all_display` (within range 1 - 255), where the user expects a refresh delay.